brain/detail/circuit.h

```diff
@@ -270,6 +271,13 @@ class Circuit::Impl
             _synapsePositions[i]->reset(
                 new brion::Synapse( _synapseSource.getPath() + (afferent ?
                       afferentPositionsFilename : efferentPositionsFilename )));
+
+        if (_synapsePositionColumns == 0)
```

Comment:
style: spaces inside (), use {} for if() statement longer than one line, or maybe define
const auto& synapse = *_synapsePositions[i];
before these lines